conf/pasteurMaestroNormal.config

```diff
@@ -0,0 +1,26 @@
+params {
+  config_profile_description = 'Institut Pasteur Maestro cluster profile'
+  config_profile_contact = 'Pierre Luisi (@pierrespc)'
+}
+
+singularity {
+  enabled = true
+  autoMounts = true
+  runOptions =  '--home $HOME:/home/$USER --bind /pasteur'
+}
+
+process {
+  executor = 'slurm'
+  scratch=false
```

Comment:
```suggestion
  scratch = false
```

Reply:
Thanks for this and for the advice for one unique profile, I'll change it next week!